File: VB/E591/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Design
Imports DevExpress.XtraPrinting

Namespace E591
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			AddHandler printableComponentLink1.CreateReportFooterArea, AddressOf printableComponentLink1_CreateReportFooterArea
		End Sub

		Private Sub printableComponentLink1_CreateReportFooterArea(ByVal sender As Object, ByVal e As CreateAreaEventArgs)
			Dim brick As TextBrick = e.Graph.DrawString("My Report Footer", Color.Navy, New RectangleF(0, 10, 400, 40), BorderSide.None)
			brick.BackColor = Color.White
			brick.Font = New Font("Tahoma", 16)
			brick.StringFormat = New DevExpress.XtraPrinting.BrickStringFormat(StringAlignment.Center)
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			 printableComponentLink1.CreateDocument()
			 printableComponentLink1.ShowPreview()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim V As New XViewsPrinting(gridControl1)
		End Sub
	End Class
End Namespace

File: CS/E591/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Design;
using DevExpress.XtraPrinting;

namespace E591
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            printableComponentLink1.CreateReportFooterArea+=new CreateAreaEventHandler(printableComponentLink1_CreateReportFooterArea);
        }

        void printableComponentLink1_CreateReportFooterArea(object sender, CreateAreaEventArgs e)
        {
            TextBrick brick = e.Graph.DrawString("My Report Footer", Color.Navy, new RectangleF(0, 10, 400, 40), BorderSide.None);
            brick.BackColor = Color.White;
            brick.Font = new Font("Tahoma", 16);
            brick.StringFormat = new DevExpress.XtraPrinting.BrickStringFormat(StringAlignment.Center);            
        }

        private void button1_Click(object sender, EventArgs e)
        {
             printableComponentLink1.CreateDocument();
             printableComponentLink1.ShowPreview();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            XViewsPrinting V = new XViewsPrinting(gridControl1);
        }
    }
}
